src/proto/hub_event.proto

```diff
@@ -59,4 +59,5 @@ message HubEvent {
     //    MergeStorageAdminRegistryEventBody merge_storage_admin_registry_event_body = 10;
     MergeOnChainEventBody merge_on_chain_event_body = 11;
   };
+  optional uint64 block_number = 12;
```

Comment:
Doesn't have to be optional, it will default to 0

Reply:
Addressed